web/modules/custom/dept_publications/dept_publications.module

```diff
@@ -122,3 +123,27 @@ function dept_publications_entity_access(EntityInterface $entity, $operation, Ac
     }
   }
 }
+
+/**
+ * Implements hook_entity_operation_alter().
+ */
+function dept_publications_entity_operation_alter(array &$operations, EntityInterface $entity) {
```

Comment:
Belts 'n' braces here. We're aiming to cover someone trying to view the node directly, as well as any operations that might be defined for it, as well as mediate in the origins workflow access control for changing to - specifically - published moderation state.